src/main/java/uk/gov/digital/ho/hocs/casework/application/LogEvent.java

```diff
@@ -1,153 +1,154 @@
 package uk.gov.digital.ho.hocs.casework.application;
```

Comment:
Is this change just reordering the list?

Reply:
`CASE_TAG_CONFLICT` has been added, apart from that yes